src/Ocelot/DownstreamUrlCreator/Middleware/DownstreamUrlCreatorMiddleware.cs

```diff
@@ -150,7 +153,8 @@ private static bool ContainsQueryString(DownstreamPath dsPath)
 
         private static bool ServiceFabricRequest(IInternalConfiguration config, DownstreamRoute downstreamRoute)
         {
-            return config.ServiceProviderConfiguration.Type?.ToLower() == "servicefabric" && downstreamRoute.UseServiceDiscovery;
+            return config.ServiceProviderConfiguration.Type?.ToLower() == "servicefabric" &&
+                   downstreamRoute.UseServiceDiscovery;
```

Comment:
Zero changes!
Revert this fake change plz!

Reply:
Fixed in commit https://github.com/ThreeMammals/Ocelot/pull/1348/commits/3c7781e7ceccf0a54e1feae91a07c78e12bcfa3d ✔️